app/controllers/ConfirmTransferController.scala

```diff
@@ -45,30 +46,51 @@ class ConfirmTransferController @Inject()(val controllerComponents: SecurityComp
       }
   }
 
+  private def loadStandardPageBasedOnCtStatus(consignmentId: UUID, httpStatus: Status,
+                                              finalTransferForm: Form[FinalTransferConfirmationData] = finalTransferConfirmationForm)
+                                             (implicit request: Request[AnyContent]): Future[Result] = {
+    val consignmentStatusService = new ConsignmentStatusService(graphqlConfiguration)
```

Comment:
Should pass the `ConsignmentStatusService` as a class constructor param, the same as the other services